lnrpc/walletrpc/walletkit.proto

```diff
@@ -400,6 +413,19 @@ message ListAccountsResponse {
     repeated Account accounts = 1;
 }
 
+message ListAddressesRequest {
+    // An optional filter to only return accounts matching this name.
+    string name = 1;
+}
+
+message ListAddressesResponse {
+    repeated string address_list = 1;
+}
+
+// message ListAddressesResponse {
+//     repeated AddressesResponseType address_list = 1;
```

Comment:
The idea is to use this message in the end, right? Anything that didn't work here?